Packs/EDL/Integrations/EDL/EDL.yml

```diff
@@ -82,6 +83,8 @@ configuration:
   type: 9
   section: Connect
   required: false
+  hidden:
+  - marketplacev2
```

Comment:
Have you tested this?
Last time I checked this wasn't showing as expected.

Reply:
tested in xsoar saas:
<img width="582" alt="image" src="https://github.com/demisto/content/assets/46249224/4da6d96b-68e8-4b92-927e-13b8a7279ece">